helm/providers/charts/cloudant/README.md

```diff
@@ -0,0 +1,47 @@
+<!--
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+-->
+
+# Use Helm to deploy Cloudant provider and package
+
+This chart is to deploy Cloudant provider and package to OpenWhisk on a Kubernetes using Helm.
+
```

Comment:
Perhaps add a short explanation that this will deploy the Cloudant package/provider to use the same Cloudant instance that is being used by OpenWhisk itself.  We may also want to add some documentation (perhaps in a followup PR) for how to deploy the provider/package to connect to a different Cloudant instance.